server/channels/api4/apitestlib.go

```diff
@@ -466,7 +463,8 @@ func (th *TestHelper) InitLogin() *TestHelper {
 	th.BasicUser2 = userCache.BasicUser2.DeepCopy()
 
 	users := []*model.User{th.SystemAdminUser, th.TeamAdminUser, th.BasicUser, th.BasicUser2, th.SystemManagerUser}
-	mainHelper.GetSQLStore().User().InsertUsers(users)
+	err := mainHelper.GetSQLStore().User().InsertUsers(users)
+	require.NoError(th.TB, err)
```

Comment:
That check is failing:
```
    apitestlib.go:467: 
        	Error Trace:	/mattermost/server/channels/api4/apitestlib.go:467
        	            				/mattermost/server/channels/api4/apitestlib.go:244
        	            				/mattermost/server/enterprise/metrics/metrics_test.go:94
        	Error:      	Received unexpected error:
        	            	pq: duplicate key value violates unique constraint "users_pkey"
        	Test:       	TestPluginMetrics
```

I wonder if we should remove the user creation cache. Password hashing no longer seems like a huge concern.